### ISSUE TITLE
Viewing key request for a secret shows blank popup

### ISSUE THREAD

**Issue by tdiesler** [chainapsis/keplr-wallet#685]
**Describe the bug**

When requesting a viewing key for a secret token, I see a white popup with no content

Keplr Extension Version (on Brave)
0.11.46

<img width="362" alt="Screenshot 2023-03-16 at 20 14 09" src="https://user-images.githubusercontent.com/244946/225729140-db497a7d-4cad-48b8-8810-f2f61c9dd004.png">

```


  | <!DOCTYPE html>
-- | --
  | <html lang="en">
  | <head>
  | <meta charset="UTF-8" />
  | <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  | <title>Keplr</title>
  | <script type="application/javascript" src="browser-polyfill.js"></script>
  | </head>
  | <body>
  | <div id="app"></div>
  | <script type="text/javascript" src="popup~popup~493df0b3.bundle.js"></script><script type="text/javascript" src="popup~popup~dc51bad7.bundle.js"></script><script type="text/javascript" src="popup~popup~12ce8da2.bundle.js"></script><script type="text/javascript" src="popup~popup~9722bd1d.bundle.js"></script><script type="text/javascript" src="popup~popup~8bc817dc.bundle.js"></script><script type="text/javascript" src="popup~popup~b66e31ff.bundle.js"></script><script type="text/javascript" src="popup~popup~5b0bcb4a.bundle.js"></script><script type="text/javascript" src="popup~popup~3565fc9e.bundle.js"></script><script type="text/javascript" src="popup~popup~47dad47d.bundle.js"></script><script type="text/javascript" src="popup~popup~55d7db5f.bundle.js"></script><script type="text/javascript" src="popup~popup~ef648799.bundle.js"></script><script type="text/javascript" src="popup~popup~d0ae3f07.bundle.js"></script><script type="text/javascript" src="popup~popup~91f4305a.bundle.js"></script><script type="text/javascript" src="popup~popup~5a11b65b.bundle.js"></script><script type="text/javascript" src="popup.bundle.js"></script></body>
  | </html>
  |  

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Keplr</title>
    <script type="application/javascript" src="[browser-polyfill.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/browser-polyfill.js)"></script>
  </head>
  <body>
    <div id="app"></div>
  <script type="text/javascript" src="[popup~popup~493df0b3.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~493df0b3.bundle.js)"></script><script type="text/javascript" src="[popup~popup~dc51bad7.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~dc51bad7.bundle.js)"></script><script type="text/javascript" src="[popup~popup~12ce8da2.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~12ce8da2.bundle.js)"></script><script type="text/javascript" src="[popup~popup~9722bd1d.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~9722bd1d.bundle.js)"></script><script type="text/javascript" src="[popup~popup~8bc817dc.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~8bc817dc.bundle.js)"></script><script type="text/javascript" src="[popup~popup~b66e31ff.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~b66e31ff.bundle.js)"></script><script type="text/javascript" src="[popup~popup~5b0bcb4a.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~5b0bcb4a.bundle.js)"></script><script type="text/javascript" src="[popup~popup~3565fc9e.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~3565fc9e.bundle.js)"></script><script type="text/javascript" src="[popup~popup~47dad47d.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~47dad47d.bundle.js)"></script><script type="text/javascript" src="[popup~popup~55d7db5f.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~55d7db5f.bundle.js)"></script><script type="text/javascript" src="[popup~popup~ef648799.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~ef648799.bundle.js)"></script><script type="text/javascript" src="[popup~popup~d0ae3f07.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~d0ae3f07.bundle.js)"></script><script type="text/javascript" src="[popup~popup~91f4305a.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~91f4305a.bundle.js)"></script><script type="text/javascript" src="[popup~popup~5a11b65b.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup~popup~5a11b65b.bundle.js)"></script><script type="text/javascript" src="[popup.bundle.js](chrome-extension://dmkamcknogkgcdfhhbddcghachkejeap/popup.bundle.js)"></script></body>
</html>```

**Comment by HeesungB** [comment]
@tdiesler 
I'm fixing it.
Thank you for the information.

**Comment by tdiesler** [comment]
This seems to be fixed in 0.11.48
